### PR TITLE
feat: remove use_pointcloud_container

### PR DESCRIPTION
**Pull request by kminoda**
## Description
See https://github.com/autowarefoundation/autoware.universe/pull/6115

## Tests performed
See https://github.com/autowarefoundation/autoware.universe/pull/6115

## Effects on system behavior

See https://github.com/autowarefoundation/autoware.universe/pull/6115

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: .vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -0,0 +1,741 @@
+{
```
Comment (miursh) [inline .vscode/c_cpp_properties.json:1]: Could you remove .vscode files?

File: autoware_launch/launch/autoware.launch.xml [hunk 2]
```diff
@@ -2,8 +2,8 @@
 <launch>
   <!-- Essential parameters -->
   <arg name="map_path" description="point cloud and lanelet2 map directory path"/>
-  <arg name="vehicle_model" default="sample_vehicle" description="vehicle model name"/>
-  <arg name="sensor_model" default="sample_sensor_kit" description="sensor model name"/>
```
Comment (miursh) [inline autoware_launch/launch/autoware.launch.xml:6]: These lines revert https://github.com/autowarefoundation/autoware_launch/pull/768. 
 Could you please remove these changes?

File: kminoda_autoware.log [hunk 3]
```diff
@@ -0,0 +1,7 @@
+vehicle name is sample
```
Comment (miursh) [inline kminoda_autoware.log:1]: Could you remove this log file?

File: .vscode/c_cpp_properties.json (hunk 1 above)
Comment (kminoda) [reply .vscode/c_cpp_properties.json]: Sorry it was mistakenly added somehow :bow: 
Fixed: https://github.com/autowarefoundation/autoware_launch/pull/806/commits/f8a56889dc298ef9b00d781e8e21a63b3c08f63e

File: kminoda_autoware.log (hunk 3 above)
Comment (kminoda) [reply kminoda_autoware.log]: Sorry it was mistakenly added somehow :bow: 
Fixed: https://github.com/autowarefoundation/autoware_launch/pull/806/commits/f8a56889dc298ef9b00d781e8e21a63b3c08f63e

File: autoware_launch/launch/autoware.launch.xml (hunk 2 above)
Comment (kminoda) [reply autoware_launch/launch/autoware.launch.xml]: Fixed: https://github.com/autowarefoundation/autoware_launch/pull/806/commits/9f988d86fa49b9eca4109b321cc3420116406f17

**Review by miursh — APPROVED** [review]
LGTM

**Pull request merged**